api/services/containers/v1/containers.proto

```diff
@@ -65,6 +65,9 @@ message Container {
 	// Spec to be used when creating the container. This is runtime specific.
 	google.protobuf.Any spec = 5;
 
+	// Snapshotter specifies the snapshotter name used for rootfs
+	string snapshotter = 6;
```

Comment:
I guess the only thing with this is the same as the runtime.  Are we ever going to need to specify additional options or configuration for the snapshotter, on the per container level, where this string would have to be changed to an object in the future?